guides/common/modules/proc_deploying-the-ca-certificate-on-a-host-by-using-script-rex.adoc

```diff
@@ -0,0 +1,32 @@
+[id="deploying-the-ca-certificate-on-a-host-by-using-script-rex"]
+= Deploying the CA certificate on a host by using Script REX
+
+You can use remote execution (REX) with the Script provider to deploy the CA certificate.
+
+include::snip_prerequisites-deploying-ca-cert-rex.adoc[]
+
+.Procedure
+. In the {ProjectWebUI}, navigate to *Monitor* > *Jobs*.
+. Click *Run Job*.
+. From the *Job category* list, select `Commands`.
```

Comment:
@Lennonka I realize that this has already been merged. I found this PR when I was trying to figure out the RN for this feature. UI elements and options should be marked up as bold, not with backticks.